examples/express-ts/package.json

```diff
@@ -14,24 +14,24 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.16.1",
-    "express": "^4.14.1",
-    "express-promise-router": "^1.1.1",
-    "knex": "^0.12.7",
+    "body-parser": "^1.17.2",
+    "express": "^4.15.3",
+    "express-promise-router": "^2.0.0",
+    "knex": "^0.13.0",
     "lodash": "^4.17.4",
-    "morgan": "^1.8.1",
-    "objection": "^0.7.0",
+    "morgan": "^1.8.2",
+    "objection": "../..",
```

Comment:
@flow Actually I totally disagree--especially if there's a functional discussion that happens from the comment! In almost any case, scrutiny should be encouraged. 

I thought it was a neat trick, too (and first saw it in one of @sindresorhus' repos, I think?)